examples/pytorch/graphsage/README.md

```diff
@@ -71,6 +71,20 @@ Notably,
 
 Micro F1 score reaches 0.9212 on test set.
 
+### Use GPU sampling and CUDA UVA sampling
+
+For training scripts `train_sampling.py`, `train_sampling_multi_gpu.py` and `train_sampling_unsupervised.py`, we provide arguments `--sample-device` and `--data-device`.
+
+For `--sample-device`, we provide the following choices:
```

Comment:
Also is it better to call it graph device instead of sample device?

Reply:
Agreed.